### PR TITLE
Fixed redirection after password reset

### PR DESCRIPTION
**Pull request by lal12**
Fixed error in lostpassword.js, that prevented redirecting after password reset.

I described the error here: https://github.com/nextcloud/server/issues/7218

### CODE REVIEW

**Comment by lal12** [comment]
Hmm, the test build reports failures, but most of them show a server related error message:

> Error response from daemon: could not find an available, non-overlapping IPv4 address pool among the defaults to assign to the network

File: core/js/lostpassword.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ OC.Lostpassword = {
 		var resetErrorMsg;
 		if (result && result.status === 'success'){
 			$.post(
-					OC.webroot + '/',
+					OC.webroot + '/ocs/v1.php',
```
Comment (rullzer) [inline core/js/lostpassword.js]: mmmm posting to that endpoint makes no sense... why do we actually do that? to trigger the login or what?

**Review by rullzer — CHANGES_REQUESTED** [review]

File: core/js/lostpassword.js (hunk 1 above)
Comment (lal12) [reply core/js/lostpassword.js]: I wondered about that too. I just assumed it is to check whether the password changed. But usually this error should be reported by the response from the previous request. But I am not familiar with the next loud code.

Comment (lal12) [reply core/js/lostpassword.js]: So should I remove this request instead?

Comment (rullzer) [reply core/js/lostpassword.js]: Year try it and see if it still works :)

Comment (lal12) [reply core/js/lostpassword.js]: Look here: https://github.com/nextcloud/server/pull/7691,
I accidentily deleted my fork...

Comment (rullzer) [reply core/js/lostpassword.js]: Ok lets close this then

**Pull request closed without merging**